app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -78,6 +98,8 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
                     if (available) {
                         components.analytics.metrics.track(Event.ReaderModeAvailable)
                     }
+
+                    readerModeAvailable = available
                     runIfFragmentIsAttached {
```

Comment:
Here's we'd need to call `browserToolbarView.view.invalidateActions()` to make sure the actions update. See also https://github.com/mozilla-mobile/reference-browser/blob/master/app/src/main/java/org/mozilla/reference/browser/browser/ReaderViewIntegration.kt#L67